styles/components/_modal.scss

```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
+    font-size: 2.5rem;
+  }
+
+  &__win-image {
```

Comment:
maybe case modal__image--win? We could join all the modal styles in one file later.

Reply:
done